src/include/distributed/errormessage.h

```diff
@@ -53,7 +53,8 @@ DeferredErrorMessage * DeferredErrorInternal(int code, const char *message, cons
 		RaiseDeferredErrorInternal(error, elevel); \
 		if (__builtin_constant_p(elevel) && (elevel) >= ERROR) { \
 			pg_unreachable(); } \
-	} while (0)
+	} \
+	while (0)
```

Comment:
Well, it keeps coming when I execute `citus_indent`. I'm OK to undo this on the PR. I'll do it once I rebase it.